internal/cmd/passwords/passwords.go

```diff
@@ -0,0 +1,121 @@
+	cmd.MarkPersistentFlagRequired("org") // nolint:errcheck
+
+	cmd.AddCommand(CreateCmd(ch))
+	cmd.AddCommand(ListCmd(ch))
+	cmd.AddCommand(DeleteCmd(ch))
+	// cmd.AddCommand(ShowCmd(ch))
+
+	return cmd
+}
+
+type Passwords []*Password
+
+type Password struct {
+	PublicID  string `header:"id" json:"id"`
+	Name      string `header:"name" json:"name"`
+	UserName  string `header:"username" json:"username"`
+	Role      string `header:"role" json:"role"`
+	RoleDesc  string `header:"role description" json:"-"`
+	CreatedAt int64  `json:"created_at"`
+	orig      *ps.DatabaseBranchPassword
+}
+
+type DeletedPassword struct {
+	Name      string `header:"name" json:"name"`
+	Role      string `header:"role" json:"role"`
+	CreatedAt int64  `header:"created_at,timestamp(ms|utc|human)" json:"created_at"`
+	DeletedAt int64  `header:"deleted_at,timestamp(ms|utc|human)" json:"deleted_at"`
+	orig      *ps.DatabaseBranchPassword
+}
+
+type PasswordWithPlainText struct {
+	Name              string               `header:"name" json:"name"`
+	PublicID          string               `header:"username" json:"username"`
+	AccessHostUrl     string               `header:"access host url" json:"access_host_url"`
+	Role              string               `header:"role" json:"role"`
+	RoleDesc          string               `header:"role description" json:"role_description"`
+	PlainText         string               `header:"plain text" json:"plain_text"`
+	ConnectionStrings ps.ConnectionStrings `json:"connection_strings"`
+	orig              *ps.DatabaseBranchPassword
+}
+
+func (b *Password) MarshalJSON() ([]byte, error) {
+	return json.MarshalIndent(b.orig, "", "  ")
+}
+
+func (b *Password) MarshalCSVValue() interface{} {
+	return []*Password{b}
+}
+
+func (b Passwords) String() string {
+	var buf strings.Builder
+	tableprinter.Print(&buf, b)
+	return buf.String()
+}
+
+// toPassword Returns a struct that prints out the various fields of a branch model.
+func toPassword(password *ps.DatabaseBranchPassword) *Password {
+	return &Password{
+		Name:      password.Name,
+		PublicID:  password.PublicID,
+		UserName:  password.PublicID,
+		Role:      password.Role,
+		RoleDesc:  toRoleDesc(password.Role),
+		CreatedAt: password.CreatedAt.UTC().UnixNano() / (int64(time.Millisecond) / int64(time.Nanosecond)),
+		orig:      password,
+	}
+}
+
+func toPasswords(passwords []*ps.DatabaseBranchPassword) []*Password {
+	bs := make([]*Password, 0, len(passwords))
+	for _, password := range passwords {
+		bs = append(bs, toPassword(password))
+	}
+	return bs
+}
+
+func toPasswordWithPlainText(password *ps.DatabaseBranchPassword) *PasswordWithPlainText {
+	return &PasswordWithPlainText{
+		Name:              password.Name,
+		PublicID:          password.PublicID,
+		PlainText:         password.PlainText,
+		AccessHostUrl:     password.Branch.AccessHostURL,
+		Role:              password.Role,
+		RoleDesc:          toRoleDesc(password.Role),
+		orig:              password,
```

Comment:
nit: can you move this to the bottom. We should make sure it doesn't get removed in the future accidentally.